tests/wpt/mozilla/tests/mozilla/img_width_height_not_rendered.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Image intrinsic dimensions are returned if the image isn't rendered</title>
```

Comment:
Assuming this passes in Firefox and Chrome, let's move it to web-platform-tests/html/semantics/embedded-content/the-img-element/ instead.